packages/forklift-console-plugin/src/plans/create/steps/GeneralInformationForm.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+import { FormErrorHelperText } from 'src/components/FormErrorHelperText';
+import { useForkliftTranslation } from 'src/utils';
+
+import {
+  Flex,
+  Form,
+  FormGroup,
+  FormSection,
+  TextInput,
+  Title,
+  ValidatedOptions,
+} from '@patternfly/react-core';
+
+import { GeneralFormFieldId, generalFormFieldLabels } from './constants';
+
+export const GeneralInformationForm: FC = () => {
+  const { t } = useForkliftTranslation();
+  const {
+    control,
+    formState: { errors },
+  } = useFormContext();
+
+  return (
+    <Flex
+      direction={{ default: 'column' }}
+      spaceItems={{ default: 'spaceItemsLg' }}
+      className="create-plan-wizard__form"
```

Comment:
if you already having cusotm CN , you can add space items and direction to it - so only a single place is handling styling